### PR TITLE
MINOR: Add pull request template

### PR DESCRIPTION
**Pull request by ijuma**

### CODE REVIEW

**Comment by asfgit** [comment]
FAILURE
 No test results found.
--none--

**Comment by ijuma** [comment]
This is an initial attempt, thoughts on what else we should add? cc @junrao @hachikuji @rajinisivaram @ewencp @guozhangwang @gwenshap @cmccabe @apurvam @becketqin @onurkaraman

**Comment by asfgit** [comment]
FAILURE
 No test results found.
--none--

**Comment by asfgit** [comment]
FAILURE
 7268 tests run, 1 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
FAILURE
 7268 tests run, 1 skipped, 0 failed.
--none--

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+*More detailed description of your change,
```
Comment (apurvam) [inline PULL_REQUEST_TEMPLATE.md:1]: Would we want to add a line item for suggestions on the title?

Comment (apurvam) [reply PULL_REQUEST_TEMPLATE.md:1]: This would mean adding something like *KAFKA-1234: Short summary of the change.* at the top of this file.

**Comment by guozhangwang** [comment]
@ijuma how are we going to educate submitters to follow the template?

**Comment by ijuma** [comment]
@guozhangwang the template is what appears in the pull request text box.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ijuma) [reply PULL_REQUEST_TEMPLATE.md:1]: Does the first line of the template appear in the PR title?

Comment (apurvam) [reply PULL_REQUEST_TEMPLATE.md:1]: oh. I see from your comment below that this is what the PR box would. I know the first line of the commit message appears as the PR title, but not for the actual text in the description box of the PR page. 

Is there a way to have a template for the title?

Comment (ijuma) [reply PULL_REQUEST_TEMPLATE.md:1]: I need to check, it's a good question and would be good to do it, if possible.

**Comment by becketqin** [comment]
The template looks good. Is it possible to add some check to the format and maybe have a warning comment if the submitter did not follow the rule?

**Comment by ijuma** [comment]
@becketqin what do you mean by "add some check to the format"?

**Comment by becketqin** [comment]
@ijuma For example, check if the title is in correct format, or commit message is missing. I have also seen from time to time that the commit comments has "@SOMEONE", which seems a little weird.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ijuma) [reply PULL_REQUEST_TEMPLATE.md:1]: I couldn't find a way to set the title from the template.

**Comment by ijuma** [comment]
I don't think you can have automated checks via GitHub.

**Review by apurvam — APPROVED** [review]
LGTM, thanks!

**Comment by becketqin** [comment]
Got it. The template LGTM.

**Comment by ijuma** [comment]
I'm going to go ahead and merge this. We can improve it further in subsequent PRs if necessary.

**Comment by guozhangwang** [comment]
> I'm going to go ahead and merge this. We can improve it further in subsequent PRs if necessary.

Thanks!

**Pull request closed without merging**